public/testem/testem_client.js

```diff
@@ -186,6 +186,28 @@ var Testem = {
     }
     this.evtHandlers[evt].push(callback);
   },
+  once: function(evt, callback) {
+    if (!this.evtHandlers) {
+      this.evtHandlers = {};
+    }
+    if (!this.evtHandlers[evt]) {
+      this.evtHandlers[evt] = [];
+    }
+    const callbackIndex = this.evtHandlers[evt].length;
+    let wrapper = function(){
```

Comment:
use `var` instead of `let` and `const` in the meantime, so that we are consistent with the rest of the file. I don't remember what the browser support is for testem, but anything IE10 and older doesnt support `let` or `const` at all.